infra/base-images/base-runner/bad_build_check

```diff
@@ -27,6 +27,11 @@ THRESHOLD_FOR_NUMBER_OF_EDGES=100
 ASAN_CALLS_THRESHOLD_FOR_ASAN_BUILD=1000
 ASAN_CALLS_THRESHOLD_FOR_NON_ASAN_BUILD=0
 
+# The value below can definitely be higher (like 500-1000), but avoid being too
+# agressive here while still evaluating the DFT-based fuzzing approach.
+DFSAN_CALLS_THRESHOLD_FOR_DFSAN_BUILD=100
```

Comment:
I think bad_build_check is being too complicated for bash and should be converted to python. To compensate for this, the code is quite repetitive.

Reply:
Agreed, would be nice to have a new implementation, but not in this CL :)